README.md

```diff
@@ -95,21 +80,16 @@ A curated list of anime streaming/downloading websites.
 
 * [bato.to](http://bato.to/)
 * [bulumanga.com](http://bulumanga.com/)
-* [jaiminisbox.com](https://jaiminisbox.com/)
-* [mangafox.me](https://mangafox.me/)
+* [fanfox.net](https://fanfox.net/)
 * [mangafreak.eu](http://mangafreak.eu/)
-* [mangahere.co](https://www.mangahere.co/)
+* [mangahere.cc](https://mangahere.cc/)
 * [mangakakalot.com](http://mangakakalot.com/)
-* [manganel.com](http://manganel.com)
+* [manganelo.com](https://manganelo.com//)
 * [mangapanda.com](http://www.mangapanda.com/)
-* [mangapark.me](http://mangapark.me/)
+* [mangapark.net](http://mangapark.net/)
 * [mangareader.net](http://www.mangareader.net/)
-* [mangaseeonline.us](http://mangaseeonline.us/)
```

Comment:
chnage to https://mangasee123.com/

Reply:
I have to go somewhere, I will do big implementation on this, so just wait :)

Reply:
so I will just change mangasee123.com nowness.

Reply:
feel free to change when convenient :)